proposal.rst

```diff
@@ -38,56 +38,64 @@
 .. which ideas underpin your project, and how you've had to develop them
 .. further to make them practical for Xapian.
 ..
+
+
 .. For academic research, it's helpful to include a URL if the paper is
 .. freely available online (via an author's website or preprint server,
 .. for instance). Not all Xapian contributors have free access to academic
 .. publishers. You should still provide all the normal information used
 .. when citing academic papers.
-.. 
+..
+
+
 .. You're welcome to include diagrams or other images if you think they're
 .. helpful - for how to do this see:
 .. https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#images
-.. 
+..
 .. Please take care to address all relevant questions - attention to detail
 .. is important when working with computers!
-.. 
+..
 .. If you have any questions, feel free to come and chat with us on IRC, or
 .. send a mail to the mailing lists.  To answer a very common question, it's
 .. the mentors who between them decide which proposals to accept - Google just
 .. tell us HOW MANY we can accept (and they tell us that AFTER student
 .. applications close).
-.. 
+..
 .. Here are some useful resources if you want some tips on putting together a
 .. good application:
-.. 
+..
 .. "Writing a Proposal" from the GSoC Student Guide:
 .. https://google.github.io/gsocguides/student/writing-a-proposal
-.. 
+..
 .. "How to write a kick-ass proposal for Google Summer of Code":
 .. https://teom.wordpress.com/2012/03/01/how-to-write-a-kick-ass-proposal-for-google-summer-of-code/
 
 ======================================
-FILLME WITH THE TITLE OF YOUR PROPOSAL
+Improve estimated total number of results
 ======================================
 
 About You
 =========
 
- * Name: FILLME
+ * Name: Andelrahman Atef Mohamed
 
- * E-mail address: FILLME
+ * E-mail address: boda998@yahoo.com
 
- * IRC nickname(s): FILLME
+ * IRC nickname(s): Boda
 
- * Any personal websites, blogs, social media, etc: FILLME
+ * Any personal websites, blogs, social media, etc: https://www.facebook.com/boda.sadala
 
- * github URL: FILLME
+ * github URL: https://github.com/BodaSadalla98
 
  * Biography:
 
-.. Tell us a bit about yourself.
 
-FILLME
+
+	Hey, I am Boda, Computer Engineering student from Egypt.
+	I have a great passion for Competitve programming, I have praticipated in many
```

Comment:
You might want to spell-check this (e.g. "Competitve" and "praticipated" are both typos)